### DISCUSSION TITLE
Input component bug

### DISCUSSION THREAD

**Discussion by uigywnkiub** [heroui-inc/heroui#4421] [Help]
Inside Input component cannot select text with cmd + a

**Comment by wingkwong** [comment]
which version are you using? just tested in docs and didn't see any problem.

![image](https://github.com/user-attachments/assets/c5865ac4-900b-4c5c-838c-b739e5418440)

**Reply by uigywnkiub** [reply]
> which version are you using? just tested in docs and didn't see any problem.
> 
> ![image](https://private-user-images.githubusercontent.com/35857179/397923014-c5865ac4-900b-4c5c-838c-b739e5418440.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ3Nzk5NjksIm5iZiI6MTczNDc3OTY2OSwicGF0aCI6Ii8zNTg1NzE3OS8zOTc5MjMwMTQtYzU4NjVhYzQtOTAwYi00YzVjLTgzOGMtYjczOWU1NDE4NDQwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjIxVDExMTQyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVhYjc4OWRmN2UyYWJmYmVmM2ZjZDIyMzczY2Y5OWRiOTdjMWQzOTdjMjk3NGYwNDI5ZGYwNGJlM2E4NmUzNjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.PtK2m58lLuZOcKXMasiYeXu-b6KOOBBCDcELV8mwiw8)

Oh, looks like it's my local strange error. Now it works.
